### DISCUSSION TITLE
使用DDNS-GO后v6地址可以Ping通但无法访问

### DISCUSSION THREAD

**Discussion by KiWinger** [jeessy2/ddns-go#799] [Q&A]
联合GPT和Issues还有各大搜索引擎检索相关问题，没找到解决方案，故来问问
环境：DSM6.2.4 / DDNS-GO5.5.1(矿神套件) / Cloudflare / 桥接，路由器每天晚上重启

5.4.1安装后，前一个礼拜一切正常，貌似从更新5.5.0后出现了问题，可能是被修改了什么系统配置文件，导致一切乱套了
- **v6地址重复记录**
(550)早期于ssh通过命令获取地址时可以列出十个以上，均为不带EUI-64的地址，无带EUI-64的地址；
(551)或为重复不带EUI-64+带EUI-64的地址；
fe80内网地址不会重复记录，不论在哪看都一样，重启NAS后刷新记录列表。

- **v6地址于外网可以Ping通但无法Tcping也无法访问**
(几率为更新550后)外网访问提示 ERR_CONNECTION_RESET 或 TIME_OUT 等；
内网访问公网地址一切正常，无论是带不带EUI-64都可以正常访问。

**Comment by jeessy2** [comment]
5.4.1到5.5.1就改了密码验证，使用服务运行加了二个缺失的参数，其它没动

你说的二点不会因为升级而改变，多检查下系统
